es_ES/layout.md

```diff
@@ -0,0 +1,68 @@
+
+Las restricciones a veces son "estrictas", lo que significa que no dejan espacio para que la caja en el renderizado decida un tamaño
+(e.g. si el mínimo y máximo  de ancho son el mismo, se dice que tienen un ancho estricto). El widget de ejemplo principal de esta `App`, es restringido por la clase
+[`RenderView`](https://docs.flutter.io/flutter/rendering/RenderView-class.html): la caja usada por el hijo devuelto por la función principal de la aplicación [`build`](https://docs.flutter.io/flutter/widgets/State/build.html) le da una restricción que lo fuerza a llenar exactamente el área del contenido de la aplicación  (comúnmente toda la aplicación). Muchas de las cajas en Flutter, especialmente aquellas que tienen un único hijo, van a pasar sus restricciones a sus hijos. Eso significa que si anida un grupo de cajas dentro de otra en la raíz del árbol de renderizado de tu aplicación, todas encajarán exactamente entre sí, forzadas por las restricciones estrictas.
+
+Algunas cajas _pierden_ las restricciones, significa que el valor máximo es mantenido pero que el mínimo es removido. Por ejemplo, [`Center`](https://docs.flutter.io/flutter/widgets/Center-class.html).
+
+Restricciones Ilimitadas
+------------------------
+
+En ciertas situaciones, las restricciones dadas por la caja pueden ser _ilimitadas_, o infinitas. Estos significa que el valor máximo de ancho o el máximo de alto es colocado a `double.INFINITY`.
+
+
+Una caja que trata de ser tan grande como pueda, no tendrá una función útil cuando se le dan restricciones ilimitadas, y en modo chequeado, esa combinación va a lanzar una excepción que apunta a ese archivo.
+
+Los casos más comunes en los que se encuentra una caja de renderizado con restricciones ilimitadas están dentro de las cajas flexibles ([`Row`](https://docs.flutter.io/flutter/widgets/Row-class.html)
+y [`Column`](https://docs.flutter.io/flutter/widgets/Column-class.html)), 
+y **dentro de regiones desplazables** 
+([`ListView`](https://docs.flutter.io/flutter/widgets/ListView-class.html) 
+y otras subclasses de [`ScrollView`](https://docs.flutter.io/flutter/widgets/ScrollView-class.html)).
+
+En particular, [`ListView`](https://docs.flutter.io/flutter/widgets/ListView-class.html)
+trata de expandirse para tratar de encajar en el espacio disponible en su dirección transversal (i.e. Si es un bloque de desplazamiento vertical, tratara de ser tan ancho como su padre). Si anida un
+[`ListView`](https://docs.flutter.io/flutter/widgets/ListView-class.html) con desplazamiento vertical dentro de un [`ListView`](https://docs.flutter.io/flutter/widgets/ListView-class.html) con desplazamiento horizontal el interno tratara de ser tan ancho como sea posible, el cual es amplio infinitamente, puesto que el externo es desplazable en esa dirección.
+
+Flexibles
+---------
+
+Las cajas comunes ([`Row`](https://docs.flutter.io/flutter/widgets/Row-class.html) 
```

Comment:
Flex boxes themselves ([`Row`](https://docs.flutter.io/flutter/widgets/Row-class.html) 
and [`Column`](https://docs.flutter.io/flutter/widgets/Column-class.html)) 
behave differently based on
whether they are in a bounded constraints or unbounded constraints in
their given direction.
Las cajas flexibles en si mismas,  ([`Row`](https://docs.flutter.io/flutter/widgets/Row-class.html) 
y [`Column`](https://docs.flutter.io/flutter/widgets/Column-class.html))  se comportan de forma diferente en función de si tienen restricciones limitadas o restricciones ilimitadas en su dirección dada.